examples/first-component/src/components/NewTodo/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import { createTodo } from "../TodoList/todos";
+import { Todo } from "../../types/Todo";
+
+interface NewTodoProps {
+  onAdd: (newTodo: Todo) => void;
+}
+
+const NewTodo: React.FC<NewTodoProps> = ({ onAdd }) => {
```

Comment:
do not import the whole React just `import { FC } from 'react'`

Reply:
Fixed